packages/ifpack2/test/unit_tests/Ifpack2_UnitTestRILUK.cpp

```diff
@@ -99,7 +99,7 @@ static Teuchos::RCP<Ifpack2::RILUK<Tpetra::RowMatrix<Scalar,LocalOrdinal,GlobalO
   //Construct a nondiagonal matrix.  It's not tridiagonal because of processor boundaries.
   const Scalar one = Teuchos::ScalarTraits<Scalar>::one();
   const Scalar two = one + one;
-  RCP<crs_matrix_type > A = Teuchos::rcp(new crs_matrix_type(map, 1));
+  RCP<crs_matrix_type > A = Teuchos::rcp(new crs_matrix_type(map, 3));
```

Comment:
Ifpack2's tests probably don't exercise actual fill, since they always use the same tridiagonal matrix.